libafl/src/executors/command.rs

```diff
@@ -29,27 +23,39 @@ use crate::{
     },
     inputs::HasTargetBytes,
     observers::{ASANBacktraceObserver, ObserversTuple, StdErrObserver, StdOutObserver},
+    std::borrow::ToOwned,
 };
+use alloc::string::String;
+use alloc::vec::Vec;
+
 #[cfg(feature = "std")]
 use crate::{inputs::Input, Error};
 
+#[cfg(all(feature = "std", unix))]
+use crate::executors::{Executor, ExitKind};
+
+#[cfg(all(feature = "std", unix))]
+use std::time::Duration;
+
+use super::HasObservers;
+
 /// How to deliver input to an external program
 /// `StdIn`: The traget reads from stdin
 /// `File`: The target reads from the specified [`InputFile`]
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub enum InputLocation {
-    /// Mutate a command line argument to deliver an input
+enum InputLocation {
+    /// Mutate a commandline argument to deliver an input
     Arg {
         /// The offset of the argument to mutate
         argnum: usize,
     },
     /// Deliver input via `StdIn`
     StdIn,
-    /// Deliver the input via the specified [`InputFile`]
+    /// Deliver the iniput via the specified [`InputFile`]
```

Comment:
typo

Reply:
The typos are still there :)
Probably from Andrea?

Reply:
No I think these typos were there and then we fixed them in main. I think ale just took the file from the fork used in the paper in which the typos were not already fixed